djoser/pipelines/user_create.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+
+from djoser import exceptions, signals
+from djoser.conf import settings
+from djoser.pipelines.base import BasePipeline
+
+User = get_user_model()
+
+
+def serialize_request(request, context):
+    serializer_class = settings.SERIALIZERS.user_create
+    serializer = serializer_class(data=request.data)
+    if not serializer.is_valid(raise_exception=False):
+        raise exceptions.ValidationError(serializer.errors)
```

Comment:
These three lines are repeated many many times. I wonder if it is ok to extract that to separate function named like `validate_serializer(request, serializer, context=None)` or the readability will suffer.

Reply:
Yup, I'm aware of the code not being DRY, however I wonder if having a flat structure and simple point of reference without redirecting code flow is not worth breaking DRY in case of pipelines. I've thought about doing it, exactly the way you have described it, but I've mixed feelings about it.